easybuild/easyblocks/n/nwchem.py

```diff
@@ -199,6 +208,9 @@ def configure_step(self):
             libmpi = "-lmpichf90 -lmpich -lopa -lmpl -lrt -lpthread"
         else:
             raise EasyBuildError("Don't know how to set LIBMPI for %s", mpi_family)
+        if self.cfg['armci_network'] in ["OPENIB"]:
+            libmpi += " -libumad -libverbs -lpthread"
+        #if LooseVersion(self.version) <= LooseVersion("6.5"):
```

Comment:
Thats what their docco says, unfortunately that probably destroys the ARMCI fix, so i haven't taken the time to test that.